tests/foreman/installer/test_installer.py

```diff
@@ -1586,3 +1586,50 @@ def test_installer_timestamp_logs():
 
     :CaseAutomation: NotAutomated
     """
+
+
+@pytest.mark.destructive
+def test_installer_pub_directory_accessibility(default_sat):
+    """Verify the public directory accessibility after disabling it from the custom-hiera
+
+    :id: 2ef78840-098c-4be2-a9e5-db60f16bb803
+
+    :steps:
+        1. Add the foreman_proxy_content::pub_dir::pub_dir_options:"+FollowSymLinks -Indexes"
+            in custom-hiera.yaml file.
+        2. Run the satellite-installer.
+        3. Access the https://SATELLITE_SERVER_NAME/pub
+
+    :expectedresults: After the custom_hira.yaml settings, the public directory should not be
+       accessible.
+
+    :CaseImportance: High
+
+    :CaseLevel: System
+
+    :BZ: 1960801
+
+    :customerscenario: true
+    """
+    custom_hira_settings = (
+        'foreman_proxy_content::pub_dir::pub_dir_options: ' '"+FollowSymLinks -Indexes"'
+    )
+    custom_hiera_location = "/etc/foreman-installer/custom-hiera.yaml"
+    ssh.download_file(
```

Comment:
@mshriver 
If I am not mistaken, I would say that we should to use `sftp_read` and `sftp_write` methods from broker for new test cases.

Reply:
Absolutely, the ContentHost/broker methods for ssh/scp/sftp should be used.

Reply:
At this point I wouldn't consider it a blocker, especially with a customer scenario test case. But I would put a TODO here, and in general would recommend using the new methods whenever possible.

Reply:
Thanks, @ogajduse @mshriver,  same has been addressed.